Modules/_json.c

```diff
@@ -1493,6 +1493,8 @@ encoder_listencode_obj(PyEncoderObject *s, _PyAccu *acc,
     /* Encode Python object obj to a JSON term */
     PyObject *newobj;
     int rv;
+    PyObject *mo = PyImport_ImportModule("collections");
```

Comment:
Wouldn't you need to decrease the reference counter to `mo` and `deque_type` after using them?